### PR TITLE
had minor issues with execution of this notebook.

### PR DESCRIPTION
**Pull request by sjanssen2**
1) conversion from oridinal to numerical was not carried out
2) missing import for ward clustering
3) added some status information while doing heavy compute stuff

### CODE REVIEW

**Comment by sjanssen2** [comment]
@wasade could you please review. I need this as a basis before I add in the ~alpha div~ bloom column

File: ipynb/Metadata cross correlation.ipynb [hunk 1]
```diff
@@ -595,7 +573,9 @@
     "    if column in clean_up:\n",
     "        ordinal_replacement[column] = ordinal_assignment(exp, clean_up[column])\n",
     "    else:\n",
-    "        ordinal_replacement[column] = ordinal_assignment(exp)"
+    "        ordinal_replacement[column] = ordinal_assignment(exp)\n",
+    "    # SMJ: do the actual conversion of data. Make sure that case does not matter!\n",
+    "    map_[column] = map_[column].apply(lambda x: x.lower() if type(x) == str else x).replace(list(map(str.lower, ordinal_replacement[column].keys())), ordinal_replacement[column].values())"
```
Comment (wasade) [inline ipynb/Metadata cross correlation.ipynb]: this is quite dense, would it be possible to refactor?

**Review by wasade — CHANGES_REQUESTED** [review]
One comment

File: ipynb/Metadata cross correlation.ipynb (hunk 1 above)
Comment (sjanssen2) [reply ipynb/Metadata cross correlation.ipynb]: I tried to make the code more explicit but found that it was a huge blow up + risk that dict order might get lost. Therefore, I decided to "just" add some comments to better explain what is going on. Is that OK with you?

**Pull request merged**